MediaProcessor/tests/ConfigFileCreator.h

```diff
@@ -0,0 +1,31 @@
+#ifndef CONFIGFILECREATOR_H
+#define CONFIGFILECREATOR_H
+
+#include <fstream>
+#include <nlohmann/json.hpp>
+#include <stdexcept>
+
+namespace MediaProcessor {
+/// @brief Create a temporary configuration file from a JSON object.
```

Comment:
Nice to see this! I was planning to start using the doxy comment strings in the core. So, good to start with the new code. Please use the `/**/` version though.